src/pages/docs/tutorials/tutorial-kitten-browser/lists/index.md

```diff
@@ -141,21 +141,25 @@ const {a: myA, b} = obj; // == const myA = obj.a, b = obj.b;
 ```
 As you saw above, you can also destructure an object received as an argument to a function:
 ```js
-url: ({index, size}) => {	// == url: (props) => { const index = props.index, size = props.size;
+url: ({index, size}) => {
+	// == const {index, size} = props;
+	// == const index = props.index, size = props.size;
 	return `//loremflickr.com/${size}/${size}/kitten?random=${index}`;
 }
 ```
+`index` prop is the index of kitten from [`Repeater` in `List` component](./#repeater). The function argument is a object which contains props of `Kitten` component, so `index` and `size` props in the function argument is destructured to the local variables named `index` and `size`.
+
```

Comment:
I want to know what this phrase explains.

Reply:
I reverted this change.